source/change-streams/tests/change-streams.yml

```diff
@@ -0,0 +1,391 @@
+collection_name: &collection_name "test"
+database_name: &database_name "change-stream-tests"
+collection_name_2: &collection_name_2 "test2"
+database_name_2: &database_name_2 "change-stream-tests-2"
```

Comment:
I'd propose `collection2_name` and `database2_name` instead. When reading these identifiers, it makes it more immediately clear that this is the second subject.